lib/diplomat/key_utils.ex

```diff
@@ -0,0 +1,200 @@
+defmodule KeyUtils do
```

Comment:
KeyUtils module is for convert a struct Key to urlsafe

I am using Datastore with a library call NDB (this is official library of Google for Python to work on Google AppEngine), and NDB implement a method call `urlsafe` to convert a key to an unique string.

This module is port of `urlsafe` from python to Elixir